sds/src/secondary_validation/luxembourg_individual_nin_checksum.rs

```diff
@@ -0,0 +1,90 @@
+use crate::secondary_validation::{LuhnChecksum, Validator, VerhoeffChecksum};
+
+pub struct LuxembourgIndividualNINChecksum;
+
+const LUXEMBOURG_INDIVIDUALS_NIN_LENGTH: usize = 13;
+
+impl Validator for LuxembourgIndividualNINChecksum {
+    fn is_valid_match(&self, regex_match: &str) -> bool {
+        /*
+         * Luxembourg uses a 13-digit identification code,
+         * consisting of the birth date formatted as YYYYMMDD followed by a number XXX
+         * ensuring persons born on the same date have a unique national ID,
+         * then a first check on YYYYMMDDXXX using the Luhn10 algorithm,
+         * and finally a check on YYYYMMDDXXX using the Verhoeff algorithm.
+         */
+        if regex_match.len() != LUXEMBOURG_INDIVIDUALS_NIN_LENGTH {
+            return false;
+        }
+
+        let verhoeff_digit = regex_match.chars().last().unwrap();
+        let luhn_digit = regex_match.chars().nth(regex_match.len() - 2).unwrap();
```

Comment:
Use `https://doc.rust-lang.org/std/iter/trait.DoubleEndedIterator.html#method.nth_back` here, which should be a bit more efficient than checking form the front. As an alternative, you could do an upfront check to ensure the entire match is `ASCII` chars, then do direct indexing into the string (since all ASCII are single-byte).